templates/docker/docker-compose.production.example.yml

```diff
@@ -0,0 +1,19 @@
+# Production Docker Setup Node
+# Redis
+# redis:
+#   image: redis
+
+# Postgres
+# db:
+#   image: postgres
+
+web:
+  build: .
+  command: npm start
+  # environment:
+  #   REDIS_HOST: redis
+  # links:
+  #   - redis
+  #   - db
+  ports:
+    - "80:3000"
```

Comment:
I feel like you would want to use the built-in ansible docker stuff over docker compose. Won't they do the same thing?